core/unit_test/TestViewMapping_a.hpp

```diff
@@ -1025,13 +1025,16 @@ void test_view_mapping() {
     Kokkos::parallel_reduce(
         Kokkos::RangePolicy<host_exec_space>(0, 10),
         KOKKOS_LAMBDA(int, int& e) {
-          // an unmanaged copy.  When the parallel dispatch accepts a move for
-          // the lambda, this count should become 1.
+          // for parallel_reduce we copy the functor into a combined
+          // functor-reducer object (with reference-counting on) before
+          // constructing the ParallelReduce object (with reference-counting
+          // turned off). When the parallel dispatch accepts a move for the
+          // lambda, this count should become 2.
```

Comment:
For the record I was the only one bothered by this at the developer meeting so I will just look the other way